readline.js

```diff
@@ -29,8 +29,10 @@ var readLine = module.exports = function(file, opts) {
   .on('data', function(data) {
      for (var i = 0; i < data.length; i++) {
         if (data[i] == 10 || data[i] == 13) { // Newline char was found.
-          lineCount++;
-          if (lineLength) emit(lineCount, byteCount);
+          if (data[i] == 10) {
```

Comment:
Hi,

because we only want to count the #10, in *nix files is the only line terminator but in windows text files the lines end with CRLF - #13#10. If we count both codes will duplicate the counter.

The other change is to emit the line event also with blank lines.